### PR TITLE
Functional tests - Add test to verify login BO

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add test to verify login BO(invalid credentials, invalid email, invalid password)
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | TEST_PATH="functional/BO/00_login/02_login.js" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24016)

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/00_login/02_login.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    await testContext.addContextItem(this, 'testIdentifier', 'openAuthenticationPage', baseContext);
+
+    await loginPage.goTo(page, global.BO.URL);
+
+    const pageTitle = await loginPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(loginPage.pageTitle);
+  });
+
+  it('should enter an invalid credentials', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'entreInvalidCredentials', baseContext);
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/BO/00_login/02_login.js]:
```suggestion
    await testContext.addContextItem(this, 'testIdentifier', 'enterInvalidCredentials', baseContext);
```

Comment (NeOMakinG) [reply tests/UI/campaigns/functional/BO/00_login/02_login.js]: And same above? Don't you think?

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/00_login/02_login.js]: Thanks! done :+1:

**Review by boubkerbribri — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**